doc/build_apps/create_vm.rst

```diff
@@ -4,8 +4,9 @@ Create Azure SGX VM
 :term:`Azure Confidential Compute` (ACC) offers SGX-enabled `DC`_-series VMs, which can be deployed like other Azure SKUs.
 Note that `DC`_ SKUs can only be configured as `Gen2`_ VMs, and so only `Gen2`_ compatible VM images can be used.
 
-.. note:: If you use `Visual Studio Code <https://code.visualstudio.com/>`_, you can install and set up the `Remote - SSH <https://code.visualstudio.com/docs/remote/ssh-tutorial>`_ extension to connect to your SGX-enabled VM.
+Your ACC VM must be deployed with Ubuntu 16.04 or 18.04 and then upgraded to Ubuntu 20.04 using `do-release-upgrade` before CCF can be installed.
```

Comment:
That's not correct, setting up a VM with Ubuntu 20.04 out of the box works fine, this is the setup everyone working on the project and the CI executors use.

Reply:
This wasn't my experience whilst creating an Azure CC VM - I have the choice of either Ubuntu 16/18 (and that is all).

Reply:
Thank you for pointing out that this flow is unnecessarily restricted, I have reported it to the right person, and it will be addressed soon. Using the generic VM creation flow should work fine.